external/radiation/radiation/radiation_driver.py

```diff
@@ -379,7 +379,7 @@ def _GFS_radiation_driver(
             raise FileNotFoundError(f"effr_in = True Not implemented")
 
         if not (gfs_physics_control.lsswr or gfs_physics_control.lslwr):
-            return
+            return {}
```

Comment:
Would it be cleaner to do this check one level up? Currently there is a somewhat confusing coupling via the check for `len(out) == 0` after the driver call. I'm also fine to leave here if you think it's better.

Reply:
Sure. Fortran invokes the driver even when it's not going to run the computations and then promptly exits (which is where this came from), but I don't think there's any reason to need to follow that pattern here.